fluidefi/fluidefi-caspernet-aggregator-tools
# Deploy parser

**Summary of change:**
- Added a new micro-service, deploy parser, which takes a fetched deploy and parse in a specific format, so it can be saved in the datasource. 

**List of tasks:** 
- [x] Define types.
- [x] Define interfaces.
- [x] Define class.
- [x] Create fixture. 
- [x] Generate the docs. 

**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/deploy-parser`
```bash
git checkout feature/deploy-parser
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm deployFetcher test has passed.
- Run the docs using live server and make sure it have been updated.
